graphql-dgs-codegen-core/src/main/kotlin/com/netflix/graphql/dgs/codegen/generators/java/DataTypeGenerator.kt

```diff
@@ -54,8 +54,8 @@ class DataTypeGenerator(private val config: CodeGenConfig, private val document:
         fieldDefinitions = if (config.generateInterfaces) {
             definition.fieldDefinitions
                     .filterSkipped()
-                    .map { Field(it.name, typeUtils.findReturnType(it.type, true), isInterfaceType = true) }
-                    .plus(extensions.flatMap { it.fieldDefinitions }.filterSkipped().map { Field(it.name, typeUtils.findReturnType(it.type, true)) })
+                    .map { Field(it.name, typeUtils.findReturnType(it.type, true), overrideGetter = true) }
```

Comment:
overrideGetter seems to indicate that this intends to add override to the getter only. Is that correct?

Reply:
Yes that's correct. We only need to apply `@Override` to the getter method (as the interface will only have the getter; and the generated implementation class has the property, getter and setter). As alternative we could name that argument `implementsInterface` so it is more general, but currently it's only needed for the getter.